stable/insights-agent/values.yaml

```diff
@@ -329,3 +329,18 @@ test:
     repository: "python"
     tag: "3.6"
     pullPolicy: "Always"
+
+awscosts:
+  enabled: false
+  schedule: "0/10 * * * *"
```

Comment:
This will run every 10 minutes - I think we want `rand * * * *`

Reply:
done